docs/API.md

```diff
@@ -1353,6 +1360,22 @@ type StatsCount struct {
 
 If the refresh boolean parameter is non-zero, the latest stats will be returned without caching.
 
+#### GET *id*/meta/supported-series
+
+The `supported-series` path returns the OS series that are defined by the entity as
```

Comment:
s/entity/charm/ ?

I think I'd call this endpoint charm-supported-series, as it's charm-specific,
and like other charm-specific endpoints, it'll return a metadata-not-found error when invoked on bundles.